### ISSUE TITLE
[Site Request] Mangazone

### ISSUE THREAD

**Issue by saqlainartazz** [manga-download/hakuneko#2289]
**Name of the website**
Mangazone

**Website urls (examples below)**
- Site: https://mangazone.cc/
- Manga List: https://mangazone.cc/alphabet/
- Manga example: https://mangazone.cc/manga/purgatory-rebirth-17650.html
- Chapter online viewer example: https://mangazone.cc/chapter/ch-1-405791.html

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

**Comment by 27136773** [comment]
Template: WordPressMadara

**Comment by S-Artaz** [comment]
when will this site be added sir?

**Comment by S-Artaz** [comment]
C:\Users\Artaz\AppData\Local\hakuneko-desktop\cache\mjs\connectors
can we make our own connectors if we know the templates?

**Comment by saqlainartazz** [comment]
Sir i went to C:\Users\Artaz\AppData\Local\hakuneko-desktop\cache\mjs\connectors and made mangazone.mjs file. I copied the content from the mjs file of mangakomi which is also based on WordPressMadara template.  Then changed the super.id to mangazone, super.label to MangaZone and this.url to https://mangazone.cc. I saved the file as MangaZone.mjs. 
Then i restarted HakuNeko and the connector showed up but when i refresh manga list it gives error: 
Failed to Update manga list for Mangazone, failed to recieve content from "https://mangazone.cc/wp-admin/admin.ajax.php" (status 404) -

**Comment by saqlainartazz** [comment]
you added the websites 2 hours ago , i dont have them yet?

**Comment by 27136773** [comment]
There is a pending pull request which has to be approved first.
Furthermore, the ticket was still open which means nothing has been integrated yet.

And please be more patient and not so pushy, instead be glad that the website is going to be added so fast.
This is rude towards users who are waiting for month/years for their website suggestion being added and not even complained once.

**Comment by saqlainartazz** [comment]
sorry, but i thought i had some problem with my client ☹
